### PR TITLE
update with latest versions for activemq, jmeter & mybatis

### LINKED ISSUE #600: Feature: Update candidate versions
Update candidate versions to the latest available for 

1. activemq
2. jmeter
3. mybatis

### PR DESCRIPTION
**Pull request by xshyamx**
Candidate version updates  and setting default version to the latest available for

1. activemq
2. jmeter
3. mybatis

Fixes #600

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala [hunk 1]
```diff
@@ -6,79 +6,50 @@ import org.bson.Document
...
-  def migration002(implicit db: MongoDatabase): Document = {
+  def migration003(implicit db: MongoDatabase): Document = {
     List(
+      "5.17.1",
+      "5.17.0",
+      "5.16.5",
+      "5.16.4",
+      "5.16.3",
       "5.16.2",
       "5.16.1",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala:39]: It seems like 5.16.1 and 5.16.2 were already in the previous migration.
Are they missing from the database now?

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala [hunk 2]
```diff
@@ -23,33 +23,20 @@ class JMeterMigration {
...
-        version =>
-          Version(
-            candidate = "jmeter",
-            version = version,
-            url =
-              s"https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-$version.zip"
-          )
-      )
-      .validate()
+  def migration003(implicit db: MongoDatabase): Unit = {
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:30]: Sorry for nitpicking but shouldn't this be `migration005`?

File: src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala (hunk 1 above)
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala:39]: I just checked, and they are in the database. Can you remove these two lines, please?

Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/ActiveMQMigrations.scala]: Good catch. I also included 5.16.0 as a duplicate.

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala (hunk 2 above)
Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/JMeterMigration.scala]: Absolutely :). Copy paste error :(

**Review by marc0der — APPROVED** [review]
LGTM! Thanks for contributing!

**Pull request merged**